WcaOnRails/app/models/round_type.rb

```diff
@@ -20,4 +20,11 @@ def cellName
   def combined?
     %w(c d e g h).include?(id)
   end
+
+  # Returns the equivalent round_type_id with cutoff (or without cutoff)
+  def self.equivalent(round_type_id)
```

Comment:
Is this used anywhere? If so, maybe we could come up with a better name? Maybe `toggle_cutoff`?